src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/SystemCommandLine/Utils.tt

```diff
@@ -211,6 +211,10 @@ public class OptionData
             default:
                 throw new Exception("Unknown Element Type " + element.Name.LocalName);
         }
+
+        var textInfo = CultureInfo.InvariantCulture.TextInfo;
+        CamelCaseName = $"{textInfo.ToLower(InitCapsName[0])}{InitCapsName.Substring(1)}";
```

Comment:
It seems to me like this code isn't really generating a CamelCaseName but rather a name with Propercase where only the first letter is capitalized (assuming all the rest of the characters are indeed lower-case). Am I missing something? Is InitCapsName already guaranteed to be capitalized wherever else needed and we're just capitalizing the first character now?